README.md

```diff
@@ -15,3 +15,23 @@ Theia is a cloud & desktop IDE implemented in TypeScript.
 ### Running the examples
 
 You can find more details on how to run the examples [here](doc/Developing.md).
+ ### Running the electron example
+ ```
+ git clone https://github.com/TypeFox/Theia.git \
+ && cd Theia \
+ && npm install \
+ && cd examples \
+ && npm run install:theia \
+ && npm install \
+ && npm run cold:start:electron
+ ```
+### Running code coverage
+ ```
+npm run coverage
+
+### to view result
+
+firefox coverage/index.html 
```

Comment:
Also does this work at all? There is no npm script called 'coverage' in the package.json